### PR TITLE
Status and Support Level

### PR DESCRIPTION
**Pull request by Kielek**
Definitions prepared by @akubik-splunk.

### CODE REVIEW

File: specification/status_and_support_level.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+element or artifact of the component, or feature. Status definitions help set
+expectations around upcoming changes and related levels of support.
+
+| Level        | Development stage | Description                                                                                                                                                                                                                                                                                               |
+|--------------|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Stable       |                   | The implementation of rhe distribution, component, or element is complete in a defined scope and should not change as part of the current major version line unless a critical bug or security issue requires it.                                                                                         |
```
Comment (theletterf) [inline specification/status_and_support_level.md:13]:
```suggestion
| Stable       |                   | The implementation of the distribution, component, or element is complete in a defined scope and should not change as part of the current major version line unless a critical bug or security issue requires it.                                                                                         |
```

**Review by theletterf — APPROVED** [review]

**Comment by akubik-splunk** [comment]
For our own protection this file needs to contain headers included in  the source document. Especially it has to directly refer to Splunk Terms and Conditions link as main source of definitions.

File: specification/status_and_support_level.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Status and support levels of open source components
+
+**Status**: [Experimental](../README.md#versioning-and-status-of-the-specification)
+
+## Definitions of status levels
```
Comment (pellared) [inline specification/status_and_support_level.md:5]: The status levels definitions are scattered in two places (here and `specification/versioning.md`). It makes me hard to follow.
I think that there are 3 concepts and they should be described in the following order:
  - status levels and development stages of components
  - support levels depending on the component owner and status level
  - versioning (and structuring) depending on component status level
 
I propose to describe everything above in a file named  `component-status.md` or simply `status.md`. I think we can even get rid of `versioning.md` as I do not think we need to call out the specification versioning policy.

Comment (pellared) [reply specification/status_and_support_level.md:10]: Based on the conversion it should be spitted into 3 documents:
- `status.md`
- `support.md`
- `versioning.md`

In future we will also need `releasing.md` defining the component releasing requirements.
Changing the PR to draft.

**Comment by Kielek** [comment]
@akubik-splunk, I am closing this PR as a stale. Feel free to reopen when needed.
